tests/test_simple.py

```diff
@@ -254,27 +252,27 @@ def test_psql(self):
 
             # check returned values (1 arg)
             res = node.psql('select 1')
-            self.assertEqual(res, (0, b'1\n', b''))
+            self.assertEqual((0, b'1\n', b''), res)
```

Comment:
not sure need to reorder these params without serous reasons, it brings junk to git changelog.